client/account-recovery/lost-password/lost-password-form/test/index.jsx

```diff
@@ -24,22 +23,22 @@ describe( 'LostPassword', () => {
 		useFakeDom();
 
 		it( 'submit button shuold be disabled if user login is blank', function() {
-			const wrapper = mount( <LostPasswordFormComponent className="test__test" /> );
+			const wrapper = mount(
+				<LostPasswordFormComponent className="test__test"
+					userLogin={ '' }
+				/> );
```

Comment:
Totally. They don't need to spread in multiple lines. Updated by cf2902b967bd283e54354399a1222d1e80f6a7c8